subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -145,40 +145,16 @@ The last set of classes have no external or internal usages and therefore were d
 - `NoOpChangeListener`
 - `DeferredUtil`
 
-[[exec_properties]]
-=== Properties deprecated in Exec task
-
-* The link:{groovyDslPath}/org.gradle.api.tasks.Exec.html#org.gradle.api.tasks.Exec:execResult[Exec.execResult]
-property has been deprecated.
-Use the link:{groovyDslPath}/org.gradle.api.tasks.Exec.html#org.gradle.api.tasks.Exec:executionResult[Exec.executionResult]
-property instead.
-
 [[java_exec_properties]]
 ==== Properties deprecated in JavaExec task
 
 * The link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:main[main] getters and setters
 in link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html[JavaExec] task have been deprecated.
 Use the link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:mainClass[mainClass] property instead.
 
-[[report_properties]]
-=== Properties deprecated in Report
-
-* The link:{groovyDslPath}/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:enabled[Report.enabled]
-property has been deprecated.
-Use the link:{groovyDslPath}/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:required[Report.required]
-property instead.
-* The link:{groovyDslPath}/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:destination[Report.destination]
-property has been deprecated.
-Use the link:{groovyDslPath}/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:outputLocation[Report.outputLocation]
-property instead.
-
 [[compile_task_wiring]]
 === Deprecated properties in `compile` task
```

Comment:
I had to leave the below list in the upgrade guide because the deprecation itself comes from an `AbstractCompile` task while the DSL reference has a separate page for each concrete compile task.